frameworks/H2OAutoML/setup.sh

```diff
@@ -13,6 +13,8 @@ PIP install --no-cache-dir -r ${HERE}/requirements.txt
 
 if  [[ "$VERSION" = "latest" ]]; then
     h2o_package="h2o"
+elif [[ "$VERSION" = "zermelo" ]]; then
+    h2o_package="${H2O_REPO}/rel-zermelo/1/Python/h2o-3.32.0.1-py2.py3-none-any.whl"
```

Comment:
3.32.0.2 is available:
```suggestion
    h2o_package="${H2O_REPO}/rel-zermelo/2/Python/h2o-3.32.0.2-py2.py3-none-any.whl"
```
```

```